docs/pages/releases.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: page
 title: Releases
-permalink: project/releases
```

Comment:
At some point, I'd like to see these tucked in under a "project" directory. Perhaps, when the navigation menu is expandable / collapsable.

Reply:
Got your idea, I will restore the project tab with some additional info. I will also put `contributing` file under the same heading.

Reply:
Great idea about a "contributing" page.

Reply:
Added